package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "version": "4.0.3",
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=8.0.0"
```

Comment:
@aladdin-add Thanks for the reminder! 

For the team: do we care about this matching [`eslint/eslint`](https://github.com/eslint/eslint/blob/master/package.json#L139)? It looks like we treat each project independently in this regard.

Reply:
the packages that eslint depended should have a same/wider range. so I think it's fine to be as is.